logs_analyzer/app/scala/src/main/scala/com/databricks/apps/logs/ApacheAccessLog.scala

```diff
@@ -0,0 +1,42 @@
+package com.databricks.apps.logs
+
+/**
+  * An entry of Apache access log.
+  */
+
```

Comment:
Again, not a sin to write what you did, but to be consistent with the case class formatting I used, you could shift the fields so they are offset by two spaces:

```
case class ApacheAccessLog(
  ipAddress: String,
  clientIdentd: String,
  userId: String,
  dateTime: String,
  method: String,
  endpoint: String,
  protocol: String,
  responseCode: Int,
  contentSize: Long) extends Serializable {
}
```